docs/operations.md

```diff
@@ -254,6 +254,49 @@ access.
 └──────────────────┘
 ```
 
+## Plugins
+
+wfx provides a flexible (out-of-tree) plugin mechanism to extend its HTTP request processing capabilities.
+A plugin operates as a subprocess initiated and supervised by wfx, thereby allowing plugins to be written in _any_
+programming language.
+
+### Enabling Plugins
+
+To enable plugins, wfx must be built with the `plugin` tag (enabled by default) and started with the
+`--client-plugins-dir`/`--mgmt-plugins-dir` flag.
+_Any executable_ in the respective directory is considered to be a plugin (symlinks to executables work as well).
+Plugins are sorted and executed in lexicographic order.
+
+### Developing Plugins
+
+Communication between wfx and a plugin is achieved by exchanging [flatbuffer](https://flatbuffers.dev/) messages via
+stdin/stdout. The flatbuffer specification is available in the [fbs](../fbs) directory.
```

Comment:
Hm, `stderr` is also used for log messages from the plugin?

Reply:
Yes, wfx doesn't care about stderr. I've added a remark in the documentation.